c2corg_ui/templates/sidemenu.html

```diff
@@ -49,7 +49,7 @@
+      <iframe src="${request.static_url('c2corg_ui:static/affiz.html')}" width="160" height="320" frameborder="0" scrolling="no"></iframe>
```

Comment:
I think there's a very good reason but I don't remember it (I don't think I have written this code in v5).

Affiz Adserver offers several alternatives to include its banner tool: JS code (see my previous commit) or an iframe:
```
<iframe src="http://server1.affiz.net/tracking/iframe_display.php?n=323739345f343337315f3232393832_38888818f6" width="160" height="320" frameborder="0" scrolling="no"></iframe>
```

I will check with the association and see if we can get rid of the iframe.